### ISSUE TITLE
better abilities for wraith titans 

### ISSUE THREAD

**Issue by jhuddle1** [StargateInvasion/SGI#119]
something feels off about the wraith titans, the super hive should have zpm overload, slug fire were you launch higher yield plasma at a opponent like in the show, and subjugation like on the sokar titan, and culling ability along w the siren call and suicide darts.  THE quicksilver hive should have expanded hangers , a targeting system trait, and maybe shield overload or a absolute defense ability ? just my thoughts on the hive titans, the other hives are done so well that the titans feel rushed

**Comment by jeffg2k** [comment]
You are correct - they are not done.  The Wraith have a bit to go before they're release ready - balance and abilities being the main things to work on now.  Appreciate the thoughts and I'll see about adding some of those additional abilities.  Not sure about the shield one as I don't think the wraith have shields, but a similar one that works on hull regen would be cool.

**Comment by jeffg2k** [comment]
Pushed up some changes for additional Titan abilities.  Also removed all Wraith shields.  Tried to adjust some of the armor and hull, but it will take a bit of feedback to help balance out what ships are relatively too weak or too powerful.

**Comment by jhuddle1** [comment]
More than happy to provide feedback, I'll treat it it when I get home tonight. Shouldn't Todd and Micheal have zpm ablities ? Todd used zpms way more and helped develop the superhive Sent from my Sprint Samsung Galaxy Note10.
-------- Original message --------From: ElJeffe <notifications@github.com> Date: 12/30/19  9:06 PM  (GMT-08:00) To: StargateInvasion/SGI <SGI@noreply.github.com> Cc: jhuddle1 <Jhuddle.09@gmail.com>, Author <author@noreply.github.com> Subject: Re: [StargateInvasion/SGI] better abilities for wraith titans  (#119) Pushed up some changes for additional Titan abilities.  Also removed all Wraith shields.  Tried to adjust some of the armor and hull, but it will take a bit of feedback to help balance out what ships are relatively too weak or too powerful.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/StargateInvasion/SGI/issues/119?email_source=notifications\u0026email_token=AJKNXJG43ITJNH3MA4VAUI3Q3LHLRA5CNFSM4KBP5QN2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEH3XGBI#issuecomment-569864965",
"url": "https://github.com/StargateInvasion/SGI/issues/119?email_source=notifications\u0026email_token=AJKNXJG43ITJNH3MA4VAUI3Q3LHLRA5CNFSM4KBP5QN2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEH3XGBI#issuecomment-569864965",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by jhuddle1** [comment]
after playing a round in both the super hive and the quicksilver, I think
they both need dart upgrades, like have the super spawn 20, and quicksilver
30, remember the hives depend on darts as both defense and offense and as
well as culling the planets ? so they would need massive amounts of darts.
ingame at least increase the titans dart capacity especially if they got
suicide darts. Second the super hive needs not only the zpm overload
capability, but the ability to do something like a heavy salvo, the snipe
ability is a good start, however this is supposed to be the most advanced
hive to date, so it should incorporate some new and interesting weapons.
The super also was gonna go colonize earth on its own without the aid of
any other vessels, so it not only should have the culling wave ability but
some ability to take over a planet instantly. I can forgive the quicksilver
since this is a ingame model,  but the lore behind the super should make it
the most deadly ship next to the euphus titan and the anubis titan.

**Comment by jeffg2k** [comment]
We currently don't have the Wraith having a ZPM natively.  They have to find one on a planet (artifact) to gain the research.  So I've avoided using language that suggests a ZPM is powering an ability unless you want that ability dependent on an artifact.

I've maxed out the number of abilities that I think we can put on a Titan, so changes would require we swap it for an existing ability or somehow combine abilities if possible.  Will adjust the darts.

**Comment by jeffg2k** [comment]
Currently the Sokar Titan has the Repossession ability that can take over a planet.
